bundle/Factory/LegacyDataFactoryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Netgen\Bundle\InformationCollectionBundle\Factory;
+
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+use eZ\Publish\Core\FieldType\Value;
+use Netgen\Bundle\InformationCollectionBundle\Value\LegacyData;
+
+interface LegacyDataFactoryInterface
```

Comment:
I haven't seen this typehinted anywhere?

Reply:
`FieldDataFactory` implements it, and it is typehinted on the `factory` property inside `DatabaseAction`.

Reply:
But not in constructor.

Reply:
Good catch! Thanks. Fixed in https://github.com/netgen/NetgenInformationCollectionBundle/pull/76/commits/3c24d1a93932f016abd4ddfc1b92b1f8f88a3728